source/ExportUnityPackage/export_unity_package.py

```diff
@@ -1799,6 +1848,9 @@ def importer_metadata(self):
         platform_data_options["enabled"] = 1
       importer_metadata = Asset.set_cpu_for_desktop_platforms(
           importer_metadata)
+      if "Android" in platforms:
```

Comment:
Thanks for the suggestion! I added this change with a new commit. It seems like this will avoid extra work when `cpu_string` is `AnyCPU` since it would be looping through the platforms and make no changes (based on my understanding that the Android `CPU` value in `importer_metadata` appears to be always be `AnyCPU` without this change). Is this what you were thinking?